### DISCUSSION TITLE
Consulta sobre precondiciones de Ejercicio 7 - TP2

### DISCUSSION THREAD

**Discussion by vlzJuan** [INGCOM-UNRN-P1/cursada-2023#68] [Preguntas y respuestas] [answered]
Buenas tardes. Hago una consulta sobre el ejercicio descripto:

### 7. Potencia lenta
- Desarrolla una función que calcule la potencia de un número elevado a una potencia dada, sin utilizar el operador de multiplicación.

Para este caso, ¿Que condiciones usaríamos para el número y la potencia? Doy las opciones que se me ocurrieron, solo quiero saber cual usar:
Numero:
*  Positivo ó {Positivo, Negativo o Cero}
*  int ó float

Potencia:
* Positiva ó {Positiva, negativa o Cero}

¿Que precondiciones elijo para iniciar el ejercicio?

Saludos

**Comment by martinvilu** [comment, marked as answer]
La negativa es interesante, pero se complica solo con sumas y restas.

Mantengan todo en enteros

**Reply by vlzJuan** [reply]
Te hago una consulta aclaratoria adicional: ¿Consideramos que los numeros negativos pueden ser base? (Trae casos particulares para potencias pares)

**Reply by martinvilu** [reply]
No por lo mismo, es interesante pero la complejidad es bastante más elevada

**Comment by samusisto** [comment]
Perdón, me alarmé con el título de la consulta .... TP7 :))

**Reply by martinvilu** [reply]
Ahí lo cambie para evitar sustos
